apps/www/src/lib/components/docs/nav/mobile-nav.svelte

```diff
@@ -49,6 +49,13 @@
 										class="text-muted-foreground"
 									>
 										{item.title}
+										{#if item.label}
+											<span
+												class="ml-2 rounded-md bg-[#adfa1d] px-1.5 py-0.5 text-xs leading-none text-[#000000] no-underline group-hover:no-underline"
```

Comment:
Isn't `group-hover:no-underline` redundant as `no-underline` already has the same effect?

Reply:
Oh you're right. I just copied the markup from `docs-sidebar-nav-items.svelte`. I'll remove these 2 classes here.

Reply:
Whoops!